src/modules/WebglDistort/Module.js

```diff
@@ -54,51 +54,51 @@ module.exports = function DoNothing(options, UI) {
 
       var texture = canvas.texture(image);
 
-      var bbox1 = {
-          nw: {
-            x: matrix1[0],
-            y: matrix1[1]
-          },
-          ne: {
-            x: matrix1[2],
-            y: matrix1[3]
-          },
-          se: {
-            x: matrix1[4],
-            y: matrix1[5]
-          },
-          sw: {
-            x: matrix1[6],
-            y: matrix1[7]
-          }
-        },
-        bbox2 = {
-          nw: {
-            x: matrix2[0],
-            y: matrix2[1]
-          },
-          ne: {
-            x: matrix2[2],
-            y: matrix2[3]
-          },
-          se: {
-            x: matrix2[4],
-            y: matrix2[5]
-          },
-          sw: {
-            x: matrix2[6],
-            y: matrix2[7]
-          }
-        };
+      // var bbox1 = {
+      //     nw: {
+      //       x: matrix1[0],
```

Comment:
Did u mean to comment these out? Just checking!!

Reply:
Yeah, that's coz of the eslint that was added recently, these variables were unused.